src/main/java/com/google/sps/utils/QuizTimingPropertiesUtils.java

```diff
@@ -125,4 +125,29 @@ public String getNewQuestion(Entity game_entity, DatastoreService datastore) {
         return (update_game_entity.getProperty("quizQuestion")).toString();
     }
 
+    //Gives the user points if they have taken a quiz
+    public Boolean giveUserPoints(Boolean userQuizStatus, Entity currentUser, DatastoreService datastore) {
```

Comment:
This function name is a bit ambiguous. If I writing code in this code base, I might think this function could be used to give an arbitrary number of points to a user. This is really a specific function for assigning points after taking a quiz and should be named as such to reflect that.